tutorials/cp-cap-java-hana-db/cp-cap-java-hana-db.md

```diff
@@ -74,7 +74,19 @@ Log into the Cloud Foundry Environment using the Cloud Foundry CLI, as follows.
     npm install --save-dev @sap/hdi-deploy
     ```
 
-3. Create an SAP HANA service instance and implicitly push all artifacts to the database using:
+3. Add the following configuration in the file `package.json` in the `db` folder of your `bookstore` project:
```

Comment:
I think we should suggest to add this to the `.cdsrc.json` file in the root folder of the `bookstore` project.
In this scenario I don't think we have a package.json in the db folder, and usually in Java we suggest to maintain all configuration in the `.cdsrc.json` file.
To get a package.json in the db folder I guess we would need to call `cds add hana` in the project. Maybe this is generally a good idea to also add to the steps here. @chgeo What do you think?

Reply:
> I think we should suggest to add this to the .cdsrc.json file in the root folder of the bookstore project.

In general, `.cdsrc.json` is better for Java projects, true.
Adding the config to the `db/package.json` is even _wrong_ because `cds build` only considers the root-level `package.json` and `.cdsrc.json`.  We have also fixed this documentation issue in capire just recently.

> To get a package.json in the db folder I guess we would need to call cds add hana in the project.

👍

Reply:
Okay, then we should simply change it to .cdsrc.json. 
No real need to call cds add hana in that scenario, where we only use `cds deploy --to hana` directly, but no hana deployer.